### ISSUE TITLE
 "Save" action Button in the Display Settings menu is ineffective

### ISSUE THREAD

**Issue by rcmn** [lxde/lxqt#191]
lenovo w520 with nividia 1000m enable and integrate disabled in bios. 
By default the DVI monitor(laptop screen) is primary and DP port(Samsung S24C) extended. So I use Display Setting (listed in Menu as "Monitor Settings") to set DP port as Primary and turn off DVI monitor . Upon reboot or logout these settings are not saved. 

Also, removing DP connection does not trigger a fall back to DVI monitor , Or If DP plug in after boot nothing happen automatically and the use of Display Setting is necessary to set the display.

Basically the "Save" action Button in the Display Settings menu is ineffective.

**Comment by jleclanche** [comment]
Closing this then? Closing this...

**Comment by pmattern** [comment]
A problem was posted, a commit was made (https://github.com/lxde/lxqt-config/commit/64ed5c3042cda99393f5aace553e038e31a2ca22), the problem got fixed. No other issue is covering the problem. How exactly is this supposed to be invalid / ...?

I mean, personally, I don't care at all. But I wonder whether this may give rise to misunderstandings. E. g. a similar problem comes up in the future, and while investigating one might think "Ah, not covered by this issue anyway." or something for no reason.
